lib-cloudsdk/pom.xml

```diff
@@ -0,0 +1,80 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>openwifi-cloudsdk</artifactId>
+  <packaging>jar</packaging>
+  <parent>
+    <groupId>com.facebook</groupId>
+    <artifactId>openwifi-base</artifactId>
+    <version>2.7.0</version>
+  </parent>
+  <properties>
+    <!-- Hack for static files located in root project -->
+    <myproject.root>${project.basedir}/..</myproject.root>
+  </properties>
+  <build>
+    <finalName>openwifi-cloudsdk</finalName>
+    <resources>
+      <resource>
+        <directory>src/main/resources</directory>
+        <includes>
+          <include>**/*</include>
+        </includes>
+      </resource>
+    </resources>
+    <plugins>
```

Comment:
Is it possible to / Is it recommended to take the common plugins and move them up into the higher level pom?

Reply:
They are already in the parent pom, that's the `pluginManagement` section (and similarly `dependencyManagement`), hence why there are only stubs in the child pom.